debian/apport-notifyd.c

```diff
@@ -23,19 +23,21 @@
 
 int checkFileExtn(const char* extn, const char* CRASH_EXTN)
 {
-	#define CRASH_EXTN ".crash"
-	size_t extnLength;
-	size_t apportLength;
+#define CRASH_EXTN ".crash"
+    size_t extnLength;
+    size_t apportLength;
 
-	extnLength = strlen(extn);
-	apportLength = strlen(CRASH_EXTN);
-	// syslog(LOG_NOTICE,"extnLength is %d and apportLength is %d\n", extnLength, apportLength);
+    extnLength = strlen(extn);
+    apportLength = strlen(CRASH_EXTN);
+    // syslog(LOG_NOTICE,"extnLength is %d and apportLength is %d\n", extnLength, apportLength);
 
-	if(apportLength > extnLength) return 1;
+    // if a file's name is shorter than ".crash",
+    // apparently it's not a crash dump 
+    if(apportLength > extnLength) return 0;
```

Comment:
That is odd. apport has never reported me of the .lock file. Have you tested this ?